ember-app/app/controllers/issue/create.js

```diff
@@ -18,6 +18,10 @@ var IssuesCreateController = Ember.Controller.extend({
   }.observes('selectedRepo'),
   otherLabels: Ember.computed.alias("selectedRepo.data.other_labels"),
   assignees: Ember.computed.alias("selectedRepo.data.assignees"),
+  //Ugly Hack for backwards Enterprise Compatibiltiy
+  assigneesEnabled: function(){
+    return this.get("selectedRepo.board.issues").isAny("assignees");
```

Comment:
The base implementation for `any` short-circuits on `found` https://github.com/emberjs/ember.js/blob/v2.6.0/packages/ember-runtime/lib/mixins/enumerable.js#L662